servers/rendering/renderer_rd/effects/copy_effects.h

```diff
@@ -132,7 +132,6 @@ class CopyEffects {
 		COPY_FLAG_FLIP_Y = (1 << 5),
 		COPY_FLAG_FORCE_LUMINANCE = (1 << 6),
 		COPY_FLAG_ALL_SOURCE = (1 << 7),
-		COPY_FLAG_HIGH_QUALITY_GLOW = (1 << 8),
 		COPY_FLAG_ALPHA_TO_ONE = (1 << 9),
```

Comment:
I guess you can change this one to keep them adjacent.

Reply:
I guess so, I did this to ensure that I didn't accidentally break anything. But since these flags are used pretty consistently, there is probably a low chance of breakage

Reply:
Done!